.github/workflows/build.yml

```
@@ -13,17 +13,15 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
-      - name: Install sonar-scanner
-        uses: sonarsource/sonarqube-github-c-cpp@v2
-        env:
-          SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }} # SonarQube URL is stored in a GitHub secret
       - name: Generate compilation database
         run: |
           mkdir build
           cmake -S . -B build
-      - name: Run sonar-scanner
+      - name: Install and run Sonar Scanner
```

we used sonar-scanner because it was the executable name.
We can also Use the SonarScanner, the name in the [Doc](https://docs.sonarsource.com/sonarqube-server/latest/analyzing-source-code/scanners/sonarscanner/). The space feels strange to me. WDYT?

```suggestion
      - name: Install and run SonarScanner
```

---

That's a fair point. 

I would remove the problem altogether and use `SonarQube Scan`, as in [tutorials](https://github.com/SonarSource/sonarcloud-webapp/pull/1279/files#diff-ab231e52fc59f450b03bef0774a4c25985b1e9dfaf9cf55636707f1ea538a4abR217) and in the [README examples](https://github.com/SonarSource/sonarqube-scan-action/blob/master/README.md?plain=1#L111).
I actually planned to do that here too, and got wrong: good you caught this up :-)

Apart from consistency, I would take `SonarQube Scan` because: 
- it matches the name of the GitHub action being used: `sonarqube-scan-action`
- with the rebranding it now works for both Server and Cloud, so we can use it everywhere
- and, most importantly, it is actually what the user want to do: scan and get result on SQ
  - the SonarScanner is just a way to achieve that, which is now not visible anymore in the yaml, as it's now hidden behind the action, whereas before the user needed to call it directly

Anyway, should I need to talk about the scanner at some point, I would use `SonarScanner CLI`, as in the Doc.

Would that find your agreement?

---

Sounds good